hs/app/reach/Main.hs

```diff
@@ -37,11 +37,36 @@ reachImages =
   ]
 
 
+--------------------------------------------------------------------------------
+printfl :: PrintfArg r => String -> r -> TL.Text
+printfl s r = TL.pack $ printf s r
+
+
+fmtInitRsh :: String
+fmtInitRsh = $(mkEmbed "init/index.rsh")
```

Comment:
If it is not toooo distracting, maybe try to make it so that we support different templates as we talked about?